### PR TITLE
Main to develop

### PR DESCRIPTION
**Pull request by LukaGiorgadze**
Main to develop

### CODE REVIEW

File: pub_login.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ if [ -z "${PUB_DEV_PUBLISH_EXPIRATION}" ]; then
   exit 1
 fi
 
-# Create credentials.json file.
+# Creat credentials.json file.
```
Comment (LukaGiorgadze) [inline pub_login.sh:26]: typo error

File: pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: http_mock_adapter
 description: HTTP mock adapter for Dio & Mockito. By simply defining requests and corresponding responses through predefined adapters, you will be able to mock requests sent via Dio.
-version: 0.1.1
+version: 0.1.2
```
Comment (LukaGiorgadze) [inline pubspec.yaml:3]: is it correct?

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ Subsequently, the date entry follows **YYYY-MM-DD** format in accordance with th
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ---
+## [v0.1.2] (2020-11-26)
```
Comment (LukaGiorgadze) [inline CHANGELOG.md:14]: is it correct?

Comment (GiorgiBeriashvili) [reply CHANGELOG.md:14]: It is, though I'd prefer to add a link to v0.1.2 in the file (at the end, just like the other versions are linked to specific commits), and also add notes for v0.1.1, but it isn't critical I guess.

File: pub_login.sh (hunk 1 above)
Comment (GiorgiBeriashvili) [reply pub_login.sh:26]: Should be fixed I think.

File: CHANGELOG.md (hunk 3 above)
Comment (LukaGiorgadze) [reply CHANGELOG.md:14]: @campfire5 ⬆️

File: pub_login.sh (hunk 1 above)
Comment (LukaGiorgadze) [reply pub_login.sh:26]: @campfire5 ⬆️

File: pubspec.yaml (hunk 2 above)
Comment (GiorgiBeriashvili) [reply pubspec.yaml:3]: Seems correct as 0.1.2 is the latest published version as far as I know.

File: pub_login.sh (hunk 1 above)
Comment (campfire5) [reply pub_login.sh:26]: file was removed forever, there was no need of this for following reason, creating credentials.json file happens inside the publish.yml

File: CHANGELOG.md (hunk 3 above)
Comment (campfire5) [reply CHANGELOG.md:14]: okay, there is no version or update for this version so there is no need of linking it, but you link it anyway if you have the desire

**Pull request merged**